src/main/java/io/stargate/sgv2/jsonapi/service/shredding/tables/RowShredder.java

```diff
@@ -58,8 +59,9 @@ public WriteableTableRow shred(TableSchemaObject table, JsonNode document) {
             .map(ColumnMetadata::getName)
             .map(
                 colIdentifier -> {
-                  if (columnValues.containsKey(colIdentifier)) {
-                    return columnValues.get(colIdentifier);
+                  Object value = columnValues.get(colIdentifier);
```

Comment:
Tiny optimization: `containsKey` already does all the work to find value (but discards it), no point in doing that AND `get()`, just get() and do cheap null check.